api/client/build.go

```diff
@@ -67,6 +67,7 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 	flCgroupParent := cmd.String([]string{"-cgroup-parent"}, "", "Optional parent cgroup for the container")
 	flBuildArg := opts.NewListOpts(opts.ValidateEnv)
 	cmd.Var(&flBuildArg, []string{"-build-arg"}, "Set build-time variables")
+	isolation := cmd.String([]string{"-isolation"}, "default", "Container isolation level")
```

Comment:
we agreed an empty string for backwards compat, this should be the same on run because if a client does not send the value it will fail.  you need to double check that and fix in this pr if its not the case.